### PR TITLE
add newer che-omnisharp-plugin 0.0.6 and...

### PR DESCRIPTION
**Pull request by nickboldt**
add newer che-omnisharp-plugin 0.0.6 and netcoredbg-theia-plugin 0.0.3

Change-Id: I5161e5061983448ddc13e3097bad53488f8fcfc0
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
Added these in CRW plugin reg; should have them upstream too

**Comment by nickboldt** [comment]
ot sure if a newer sidecar image exists (or should be built first)... just used the latest I could find

File: v3/plugins/redhat-developer/che-omnisharp-plugin/0.0.6/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/omnisharp-theia-plugin
+category: Language
+firstPublicationDate: "2020-01-05"
+spec:
+  containers:
+  - image: "quay.io/eclipse/che-sidecar-dotnet:2.2.105-2fd6e78"
+    name: theia-omnisharp
+    memoryLimit: "1024Mi"
+    args:
```
Comment (svor) [inline v3/plugins/redhat-developer/che-omnisharp-plugin/0.0.6/meta.yaml]: it is unnecessary for `quay.io/eclipse/che-sidecar-dotnet`

File: v3/plugins/redhat-developer/netcoredbg-theia-plugin/0.0.3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+firstPublicationDate: "2020-01-05"
+spec:
+  containers:
+  - image: "quay.io/eclipse/che-sidecar-dotnet:2.2.105-2fd6e78"
+    name: theia-netcoredbg
+    memoryLimit: "512Mi"
+    args:
+      - sh
+      - -c
+      - ${PLUGIN_REMOTE_ENDPOINT_EXECUTABLE}
```
Comment (svor) [inline v3/plugins/redhat-developer/netcoredbg-theia-plugin/0.0.3/meta.yaml]: it is unnecessary for `quay.io/eclipse/che-sidecar-dotnet`

File: v3/plugins/redhat-developer/che-omnisharp-plugin/0.0.6/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      - sh
+      - -c
+      - ${PLUGIN_REMOTE_ENDPOINT_EXECUTABLE}
+    volumes:
+    - mountPath: "/home/jboss/.nuget"
```
Comment (svor) [inline v3/plugins/redhat-developer/che-omnisharp-plugin/0.0.6/meta.yaml]: should be `/home/theia/.nuget`

**Pull request merged**